apps/geografiske-kodeverk-service/src/main/java/no/nav/registre/testnav/geografiskekodeverkservice/service/KodeverkService.java

```diff
@@ -45,20 +46,23 @@
     }
 
     public List<Kodeverk> getKommuner(String kommunenr) {
+        ArtifactUtils.isKommunenummer(kommunenr);
         return kommunerKodeverkListe
                 .stream()
                 .filter(kodeverk -> kommunenr == null || kodeverk.getKode().equals(kommunenr))
                 .collect(Collectors.toList());
     }
 
-    public static List<Kodeverk> getLandkoder(String land) {
-        return landkoderKodeverkListe
+    public static List<Kodeverk> getLand(String landkode) {
+        ArtifactUtils.isLandkode(landkode);
+        return landKodeverkListe
                 .stream()
-                .filter(kodeverk -> land == null || kodeverk.getNavn().equals(land))
+                .filter(kodeverk -> landkode == null || kodeverk.getKode().equals(landkode))
```

Comment:
Det er god praksis å teste string både for både null og "" verdi, det kan du gjøre ved StringUtils.isNotBlank(landkode)